Makefile

```diff
@@ -10,3 +10,6 @@ publish:
 
 test:
 	py.test -n 4 --cov hyperframe test/
+
+bench:
+	python -m pytest bench/ --benchmark-only --benchmark-group-by=name --benchmark-autosave --benchmark-compare
```

Comment:
It would be really good to move this into a tox environment, and then to have the makefile delegate to tox. Saves people the effort of managing their virtual environments.